test/integration/run.sh

```diff
@@ -25,5 +25,5 @@ else
   echo "Found no --include-slow flag, skipping the slow ones"
 fi
 
-go test -count 1 -coverpkg=./adapters/repos/... -coverprofile=coverage-integration.txt -race -tags=$tags "$@" ./adapters/repos/...
-go test -count 1 -race -tags=$tags "$@" ./usecases/classification/...
+go test -v -count 1 -coverpkg=./adapters/repos/... -coverprofile=coverage-integration.txt -race -tags=$tags "$@" ./adapters/repos/...
+go test -v -count 1 -race -tags=$tags "$@" ./usecases/classification/...
```

Comment:
I think we should remove `-v` or better maybe introduce a param for example `--verbose` that would add this `-v` if needed